src/fundus/publishers/au/west_australian.py

```diff
@@ -0,0 +1,67 @@
+import datetime
+import json
+import re
+from typing import List, Optional
+
+from lxml.etree import XPath
+
+from fundus.parser import ArticleBody, BaseParser, ParserProxy, attribute, function
+from fundus.parser.data import ArticleSection, TextSequence
+from fundus.parser.utility import (
+    generic_author_parsing,
+    generic_date_parsing,
+    generic_topic_parsing,
+    sanitize_json,
+)
+
+
+class WestAustralianParser(ParserProxy):
+    class V1(BaseParser):
+        _page_data_selector = XPath(
+            "string(//script[re:test(text(), 'window.PAGE_DATA')])",
+            namespaces={"re": "http://exslt.org/regular-expressions"},
+        )
+        _json_type_pattern = re.compile(r"^(?P<type>.*)=\s*{")
```

Comment:
This seems to be leftover. At least I couldn't find any usage in the project files.